docs/supported-languages.md

```diff
@@ -30,127 +30,47 @@ Object.entries(frontMatter).filter(
 
 This document provides information about supported languages and language maturity definitions for the following products:
 
-* Semgrep OSS Engine
```

Comment:
Are we moving this somewhere or removing it entirely? And if we are removing it entirely, can I ask why?

Reply:
Oh it's because it's not a full product! Resolving.